src/secrets-kubernetes/src/lib.rs

```diff
@@ -20,7 +20,7 @@ use std::collections::BTreeMap;
 use tracing::{error, info};
 
 const FIELD_MANAGER: &str = "materialized";
-const HARDCODED_NAME: &str = "dataflowd-secret";
+pub const SECRET_NAME: &str = "sql-mounted-secret";
```

Comment:
@alex-hunt-materialize do you have a preference for a different name here? "environment-secret" or "user-secrets" or "user-managed-secrets" or "user-secrets" all come to mind. The fact that these secrets are managed via SQL is less important IMO than the fact that they are managed by the user rather than us.

Reply:
+1 for `user-managed` as a prefix.

Reply:
Yeah, `user-managed` seems better.